templates/token-staking-dapp-template/frontend/pages/Stake/components/BenefitsOfStakingSection.tsx

```diff
@@ -0,0 +1,23 @@
+import { config } from "@/config";
+import { Image } from "@/components/ui/image";
+import { useGetTokenData } from "@/hooks/useGetTokenData";
+
+export const BenefitsOfStakingSection: React.FC = () => {
+  const { tokenData } = useGetTokenData();
+
+  return (
+    <section className="how-to-mint-container px-4 text-center max-w-screen-xl mx-auto w-full">
+      <h2 className="heading-md">Benefits of staking {tokenData?.name ?? "TOKEN"}</h2>
+
+      <ol className="flex flex-col md:flex-row items-center items-baseline md:justify-between pt-6 gap-6">
```

Comment:
vscode gave me alert saying `items-center` and `items-baseline ` have same effect, can only keep 1